README.md

```diff
@@ -38,6 +41,15 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list. Feel fr
 <br>
 > <h3> Libraries</h3>
 
+#### Loaders
+
+- [Babel Loader](https://github.com/babel/babel-loader) - Webpack plugin for Babel.
+ - Maintainer: `Babel Team`  [![Github](https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/black/png/16/github.png)](https://github.com/babel)
```

Comment:
To make things future proof, we should probably use this URL instead: `https://github.com/encharm/Font-Awesome-SVG-PNG/blob/b7f74631d79e110ef972395fc408a59bea028319/black/png/16/github.png`, or perhaps upload our own hi-res version.